### PR TITLE
Aim1

### PR DESCRIPTION
**Pull request by huqiwen0313**
Add some results for simulated single cell data

### CODE REVIEW

**Comment by huqiwen0313** [comment]
@gwaygenomics 
Hi @cgreene, I do not access to assign reviewers, can you do it ?

**Comment by cgreene** [comment]
Forgot - I created the repo so I have to give access! Gave you, @gwaygenomics , and @jaclyn-taroni write access. Once you accept that, you should be able to assign reviewers 😄

File: content/03.aim1.md [hunk 1]
```diff
@@ -29,4 +29,34 @@ We will evaluate resulting models for their applicability to rheumatic disease a
...
 ### Results
 
-TBD.
+#### VAE test on simulated single cell datasets
+#### 1. Simulation data generation: 
+Simulated single cell data was generated by splatter [@doi:10.1186/s13059-017-1305-0]. 
+
+Parameters used:
+* nCells - The number of cells to simulate: 500 - 5000
+* nGenes - The number of genes to simulate: 20000 - 6000
```
Comment (gwaybio) [inline content/03.aim1.md]: is this supposed to be `20000 - 60000`? just checking b/c `nCells` increases left to right

File: content/03.aim1.md [hunk 2]
```diff
@@ -29,4 +29,34 @@ We will evaluate resulting models for their applicability to rheumatic disease a
...
+
+Figure 1: Mean expression level of simulated genes
+
+#### 2. Visualization of simulated single cell data using VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE and PCA under different parameters
+
+  To see if different methods can recover cell types, we compared the 2D visualizations on simulated datasets. 2-layer VAE performs much better to differentiate different cell types when comparing with 1-layer VAE. The performance of VAE and t-SNE is similar and much better than PCA, but with the increase of outlier genes, 2-layer VAE is more resistant to noise.
+
+![](images/outlier.vis.png)
+
+Figure 2: 2 - layer VEA is more resistant to outlier. 2D visualization of simulated single cell data from different outlier parameters (0.1 - 0.5).
```
Comment (gwaybio) [inline content/03.aim1.md]: typo: `VAE`

File: content/03.aim1.md [hunk 3]
```diff
@@ -29,4 +29,34 @@ We will evaluate resulting models for their applicability to rheumatic disease a
...
+
+#### 3. Performance evaluation of different simulation parameters
+Clustering performance was measured by normalized mutual information (NMI).
```
Comment (gwaybio) [inline content/03.aim1.md]: Can you describe this metric a bit more?

File: content/03.aim1.md [hunk 4]
```diff
@@ -29,4 +29,34 @@ We will evaluate resulting models for their applicability to rheumatic disease a
...
+Parameters used:
+* nCells - The number of cells to simulate: 500 - 5000
+* nGenes - The number of genes to simulate: 20000 - 6000
+* nGroups - The number of celltypes: 5 - 15
+* outlier - probability of a gene that is an expression outlier:  0.1 - 0.5
+* default parameter is ncells = 600, nGenes = 20000, celltypes = 5, batchsize = 1
+
+Simulation statistics under different parameters:
+
+![](images/mean.gene.exp.bmp)
```
Comment (gwaybio) [inline content/03.aim1.md:45]: I am confused about what this figure is showing? Mean expression on Y?

**Review by gwaybio — COMMENTED** [review]
Very cool results! Interesting that the additional layer buffers noise.

It looks like this repo will store documentation only? Is there source code generating these results elsewhere?

**Comment by cgreene** [comment]
@gwaygenomics : I think this repo should store the report and link to other repos for code/data/results. We should probably get the code behind this into a repo though.

File: content/03.aim1.md (hunk 1 above)
Comment (cgreene) [reply content/03.aim1.md]: from the figure, does look like 60k

File: content/03.aim1.md [hunk 5]
```diff
@@ -29,4 +29,34 @@ We will evaluate resulting models for their applicability to rheumatic disease a
...
+
+Figure 1: Mean expression level of simulated genes
+
+#### 2. Visualization of simulated single cell data using VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE and PCA under different parameters
+
+  To see if different methods can recover cell types, we compared the 2D visualizations on simulated datasets. 2-layer VAE performs much better to differentiate different cell types when comparing with 1-layer VAE. The performance of VAE and t-SNE is similar and much better than PCA, but with the increase of outlier genes, 2-layer VAE is more resistant to noise.
```
Comment (cgreene) [inline content/03.aim1.md]: May want to do one line per sentence. Makes it easier to get comments right onto the right line 😄 . Also, you might consider showing what things look like without outliers.

File: content/03.aim1.md [hunk 6]
```diff
@@ -29,4 +29,34 @@ We will evaluate resulting models for their applicability to rheumatic disease a
...
+![](images/outlier.vis.png)
+
+Figure 2: 2 - layer VEA is more resistant to outlier. 2D visualization of simulated single cell data from different outlier parameters (0.1 - 0.5).
+
+#### 3. Performance evaluation of different simulation parameters
+Clustering performance was measured by normalized mutual information (NMI).
+
+![](images/nmi.png)
+
+Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters.
```
Comment (cgreene) [inline content/03.aim1.md]: Just to make sure - this is comparing true labels against the labels called by the algorithm, right?

**Review by cgreene — COMMENTED** [review]
a few comments/questions

File: content/03.aim1.md (hunk 1 above)
Comment (huqiwen0313) [reply content/03.aim1.md]: oh, right, it is from 20000 - 60000

File: content/03.aim1.md (hunk 4 above)
Comment (huqiwen0313) [reply content/03.aim1.md:45]: It is the mean expression of genes from different samples for different simulation parameters. I will describe it clearly. Thanks

File: content/03.aim1.md (hunk 6 above)
Comment (huqiwen0313) [reply content/03.aim1.md]: Yes. This is comparing with labels called by k-means against the true labels

**Comment by huqiwen0313** [comment]
@cgreene  and @gwaygenomics , I update the file according to your comments. Please take a look at it.

File: content/03.aim1.md [hunk 7]
```diff
@@ -29,4 +29,37 @@ We will evaluate resulting models for their applicability to rheumatic disease a
...
+
+Simulation statistics under different parameters:
+
+![](images/mean.gene.exp.bmp)
+
+Figure 1: Distribution of mean expression level of simulated genes across samples under different simulated parameters.
+
+#### 2. Visualization of simulated single cell data using VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE and PCA under different parameters
+
+To see if different methods can recover cell types, we compared the 2D visualizations on simulated datasets. <br />
```
Comment (gwaybio) [inline content/03.aim1.md]: I am confused why the `<br />` are added here? I think @cgreene 's [comment](https://github.com/greenelab/czi-hca-report/pull/1#discussion_r173277856) meant to make each sentence a new line in the markdown file, not necessarily appear as a new line in the rendered version :man_shrugging:

**Review by gwaybio — APPROVED** [review]
one comment about line breaks, otherwise LGTM

File: content/03.aim1.md (hunk 7 above)
Comment (cgreene) [reply content/03.aim1.md]: Yep - just new line in the markdown file. Not adding explicit line breaks 😄

**Comment by cgreene** [comment]
BTW on the ngenes nmi figure the accuracies are very low for tsne and single layer vae. Are you sure those are accurate? They are out of line with the accuracies everywhere else.

**Comment by huqiwen0313** [comment]
Yea, I will double check. I will update the result if there is an issue.

**Pull request merged**